atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -1220,6 +1221,41 @@ public void cleanup() {}
         assertEquals(ImmutableList.of(firstCell, secondCell), cells);
     }
 
+    @Test
+    public void testGetRowsColumnRangeMultipleIteratorsWorkSafely() {
```

Comment:
You added two tests for the `SerializableTransaction`, why only one here?

Reply:
Also, similar comment as above - do we test for the right ordering of values read?

Reply:
I changed two methods in `SerializableTransaction` but only one in `SnapshotTransaction`: `getRowsColumnRangeIterator` was already safely implemented in `SnapshotTransaction` but not in `SerializableTransaction`. That said I think I'll move the two in the AbSTT to the general AbTT, so this test shouldn't be needed any more.

In terms of value testing: yes - this is in the abstract transaction test.